projects/admin-core/src/lib/search-index/search-index-edit/search-index-edit.component.css

```diff
@@ -19,7 +19,7 @@
   margin-top: 16px;
   flex: 1;
   overflow: hidden;
-  min-height: 300px;
+  min-height: 450px;
```

Comment:
Would make a new wrapping class for the new schedule picker, I think without the overflow: hidden, then you don't need the min-height I guess.